### PR TITLE
PR: Improve Accessibility #129

### PR DESCRIPTION
**Pull request by nelsonic**
I've tried to follow the a11y guidelines outlined in #129 

image

### CODE REVIEW

File: lib/app_web/live/app_live.html.heex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="h-90 w-full font-sans">
+<main class="h-90 w-full font-sans">
```
Comment (SimonLab) [inline lib/app_web/live/app_live.html.heex:1]: We will have have two `main` tags on the document, which I don't think is a valid html markdown:
image

The other `main` tag is defined in the `live` layout: https://github.com/dwyl/mvp/blob/main/lib/app_web/templates/layout/live.html.heex

**Review by SimonLab — COMMENTED** [review]
Just one comment about the `main` tag, otherwise looks good

File: lib/app_web/live/app_live.html.heex (hunk 1 above)
Comment (nelsonic) [reply lib/app_web/live/app_live.html.heex:1]: Good catch, just checked the source: 

image

I have my editor open will remove one of them now.

Comment (nelsonic) [reply lib/app_web/live/app_live.html.heex:1]: Removed it. Thanks.

**Comment by nelsonic** [comment]
Thanks for checking. Second `<main>` tag removed from `lib/app_web/templates/layout/live.html.heex` ✅

**Review by SimonLab — APPROVED** [review]
LGTM

**Comment by nelsonic** [comment]
Merci @SimonLab 🙌

**Pull request merged**